app/stac_api/models.py

```diff
@@ -634,6 +634,8 @@ class Status(models.TextChoices):
         "This field can only be set via the API."
     )
 
+    content_encoding = models.CharField(blank=True, null=False, max_length=32, default='')
```

Comment:
we could provide the valid choices here as well, this would make it easier to integrate into the admin afterwards